src/random.c

```diff
@@ -160,10 +160,10 @@ random_get_uint64 (Random      *random)
 
     return dest;
 }
+
 /*
- * Get 32 bits of random data from the parameter 'random' object. If
- * we successfully fill in the supplied 'dest' parameter with random
- * data we return 0, on error return -1.
+ * Get 32 bits of random data from the parameter 'random' object.
+ * On error, return 0 instead of random data.
```

Comment:
same as last comment

Reply:
Reply is same as above--it is mentioned in the commit message and I expanded the text in the commit message.
FYI, the block comment is *NOT* correct and I corrected the comment.
This problem was discovered when I added -Wextra to CCFLAGS.

Reply:
Gotcha, hard to return -1 when you're returning an unsigned number :\